### PR TITLE
Initial support for Yale Doorman

### PR DESCRIPTION
**Pull request by kodsnutten**
First version of support for Yale Doorman. It's still a bit "rough around the edges", but it is certainly usable. I've left some logging in the code to be able to see responses...

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,9 +25,11 @@ let VERISURE_TOKEN = null;
 let VERISURE_INSTALLATION = null;
 let VERISURE_CALLS = {};
 let VERISURE_DEVICE_NAMES = []
+let VERISURE_DOORLOCK_CODE = null;
 
 
 const getVerisureInstallation = function(config, callback) {
+  if (config && config.doorcode) VERISURE_DOORLOCK_CODE = config.doorcode;
```
Comment (ptz0n) [inline index.js]: I don't think we need this one here.

File: index.js [hunk 2]
```diff
@@ -175,11 +191,116 @@ VerisureAccessory.prototype = {
...
+    verisure._apiClient({
+      method: 'PUT',
+      uri: `/installation/${VERISURE_INSTALLATION.giid}/device/${this.serialNumber}/${actionValue}`,
+      headers: {
+        'Cookie': `vid=${VERISURE_TOKEN}`,
+        'Accept': 'application/json, text/javascript, */*; q=0.01'
+      },
+      json: 
+      {
+          "code": VERISURE_DOORLOCK_CODE
```
Comment (ptz0n) [inline index.js]: We could just read this from `this.config.doorcode`.

**Comment by ptz0n** [comment]
Great job! Thanks for the PR, I'll have a closer look when I get home.

File: index.js (hunk 1 above)
Comment (kodsnutten) [inline index.js]: Totally agree.

File: index.js (hunk 2 above)
Comment (kodsnutten) [inline index.js]: Sure thing.

**Pull request merged**